README.md

```diff
@@ -169,6 +169,11 @@ Have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
   The examples are submitted by the user base, and can be voted up or down;
   the best entries are what people see first when they look up a command.
 
+- [kb](https://github.com/gnebbia/kb)
+  it is a minimalist command line knowledge base manager.
```

Comment:
The link becomes part of the text, so it would read `kb it is a minimalist ...`
```suggestion
  is a minimalist command line knowledge base manager.
```

Reply:
To accept @bl-ue's change, simply click the "Commit suggestion" button :-)